.env.example

```diff
@@ -19,44 +17,31 @@ CHRONOS_SCHEDULE_JOBS='0 */10 * * * *'
 CHRONOS_UPDATE_TOKENS='0 0 0 1,15 * *'
 CHRONOS_PROCESS_RECURRENCE='0 */5 * * * *'
 CHRONOS_AUTOPAUSE='0 0 5 * * *'
-CI='true'
+# Enter a short url redirect service for invitations, it needs to redirecto to /invitation-link
+# INVITATION_SHORTLINK='example.com'
+
+# AUTH
+AUTH_INTERNAL_DISABLED='false'
+AUTH_GOOGLE_DISABLED='false'
+AUTH_SSO_DISABLED='false'
+OAUTH2_REDIRECT=''
+
+# ANALYTICS
+# AMPLITUDE_WRITE_KEY='key_AMPLITUDE_WRITE_KEY'
 DD_AGENT_HOST='172.17.0.1'
 DD_API_KEY='DD_API_KEY'
 DD_APM_ENABLED='false'
 DD_TRACE_ENABLED='false'
 DD_ENV='local'
-FILE_STORE_PROVIDER='s3'
-GITHUB_CLIENT_ID='key_GITHUB_CLIENT_ID'
-GITHUB_CLIENT_SECRET='key_GITHUB_CLIENT_SECRET'
-GITHUB_WEBHOOK_SECRET='key_GITHUB_WEBHOOK_SECRET'
-GITLAB_CLIENT_ID='key_GITLAB_CLIENT_ID'
-GITLAB_CLIENT_SECRET='key_GITLAB_CLIENT_SECRET'
-GOOGLE_CLOUD_CLIENT_EMAIL='key_GOOGLE_CLOUD_CLIENT_EMAIL'
-GOOGLE_CLOUD_PRIVATE_KEY='key_GOOGLE_CLOUD_PRIVATE_KEY'
-GOOGLE_CLOUD_PRIVATE_KEY_ID='key_GOOGLE_CLOUD_PRIVATE_KEY_ID'
+# SEGMENT_WRITE_KEY='key_SEGMENT_WRITE_KEY'
+SENTRY_AUTH_TOKEN='key_SENTRY_AUTH_TOKEN'
+# SENTRY_DSN='key_SENTRY_DSN'
+
+# DB RELATED
 GRAPHQL_HOST='localhost:3000'
 GRAPHQL_PROTOCOL='http'
-HOST='localhost'
-# Enter a short url redirect service for invitations, it needs to redirecto to /invitation-link
-# INVITATION_SHORTLINK='example.com'
-# MAIL GLOBALS. PROVIDER: mailgun | google | debug | smtp
-MAIL_PROVIDER='debug'
-MAIL_FROM='key_MAIL_FROM'
-MAILGUN_API_KEY='key_MAILGUN_API_KEY'
-MAILGUN_DOMAIN='key_MAILGUN_DOMAIN'
-MAIL_GOOGLE_USER='key_MAIL_GOOGLE_USER'
-MAIL_GOOGLE_PASS='key_MAIL_GOOGLE_PASS'
-MAIL_SMTP_HOST='key_MAIL_SMTP_HOST'
-MAIL_SMTP_PORT=587
-MAIL_SMTP_USER='key_MAIL_SMTP_USER'
-MAIL_SMTP_PASSWORD='key_MAIL_SMTP_PASSWORD'
-MAIL_SMTP_USE_TLS='1' # set to '0' for false
-MAIL_SMTP_CIPHERS='HIGH:MEDIUM:!aNULL:!eNULL:@STRENGTH:!DH:!kEDH'
-MIN_LOG_DURATION='200'
-OAUTH2_REDIRECT=''
 PGADMIN_DEFAULT_EMAIL='pgadmin4@pgadmin.org'
 PGADMIN_DEFAULT_PASSWORD='admin'
```

Comment:
-1 I would make a separate section for variables which are only important for developers. These belong there.

Reply:
Done ✅